octproz/src/glwindow2d.cpp

```diff
@@ -288,24 +289,48 @@ void GLWindow2D::paintGL(){
 
 //todo: add rotational angle calculation for screenheightScaled und screenWidthScaled!
 void GLWindow2D::resizeGL(int w, int h){
-	this->screenWidthScaled = 1;
-	this->screenHeightScaled = 1;
-	if(this->keepAspectRatio){
-		float textureWidth = static_cast<float>(this->height);
-		float textureHeight = static_cast<float>(this->width);
-		float screenWidth = static_cast<float>(w);//(float)this->size().width();
-		float screenHeight = static_cast<float>(h);//(float)this->size().height();
-		float ratioTexture = textureWidth/textureHeight;
-		float ratioScreen = screenWidth/screenHeight;
-		if(ratioTexture > ratioScreen){
-			this->screenHeightScaled = (screenWidth/ratioTexture)/screenHeight;
-		}
-		else{
-			this->screenWidthScaled = (screenHeight*ratioTexture)/screenWidth;
-		}
-	}
-	//recalculate marker coordinates
-	this->slot_setMarkerPosition(this->markerPosition);
+    double deg_to_rad = 3.1415926535898/180.0;
+    double rotationAngle_rad = rotationAngle*deg_to_rad;

```

Comment:
suggestion: use float instead of double. The qt function "qDegreesToRadians" could be used to calculate rad. 
request: please use for variable names camel case convention, just to keep the code more or less consistent.